conf/reference.conf

```diff
@@ -0,0 +1,7 @@
+metrics.jvm.dispatchers.includeDefaultDispatcher = true
```

Comment:
I assume you tesed his; our local testing showed this broke under 2.6

Reply:
Please take a look at it.